test/unit/cpu/CMakeLists.txt

```diff
@@ -69,10 +69,10 @@ if (NOT MSVC)
     SOURCES test-reductions.cpp
     DEPENDS_ON gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
 
-  raja_add_test(
-    NAME test-scan
-    SOURCES test-scan.cpp
-    DEPENDS_ON gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+  #  raja_add_test(
```

Comment:
@jonesholger we could set up a kernel in the perf suite with a large number of reductions as a stress test. Thoughts?